script.js

```diff
@@ -1,5 +1,6 @@
 let projectData = [
   {
+<<<<<<< Updated upstream
```

Comment:
Merge Conflict is there, Resolve them please.
Only add your project only.

Reply:
Hi @991rajat I have resolved the conflict Please review again.